packages/components/menu/src/menu-list.tsx

```diff
@@ -10,71 +15,76 @@ export interface MenuListProps extends HTMLChakraProps<"div"> {
    * Props for the root element that positions the menu.
    */
   rootProps?: HTMLChakraProps<"div">
-  /**
-   * The framer-motion props to animate the menu list
-   */
-  motionProps?: HTMLMotionProps<"div">
 }
 
-const motionVariants: Variants = {
-  enter: {
+const enterAnim = keyframes({
+  from: {
+    opacity: 0,
+    transform: "scale(0.8)",
+    visibility: "hidden",
+  },
+  to: {
+    opacity: 1,
+    transform: "scale(1)",
     visibility: "visible",
+  },
+})
+
+const exitAnim = keyframes({
+  from: {
     opacity: 1,
-    scale: 1,
-    transition: {
-      duration: 0.2,
-      ease: [0.4, 0, 0.2, 1],
-    },
+    transform: "scale(1)",
+    visibility: "visible",
   },
-  exit: {
-    transitionEnd: {
-      visibility: "hidden",
-    },
+  to: {
     opacity: 0,
-    scale: 0.8,
-    transition: {
-      duration: 0.1,
-      easings: "easeOut",
-    },
+    transform: "scale(0.8)",
+    visibility: "hidden",
   },
-}
-
-const MenuTransition = chakra(motion.div)
+})
 
 export const MenuList = forwardRef<MenuListProps, "div">(function MenuList(
   props,
   ref,
 ) {
-  const { rootProps, motionProps, ...rest } = props
-  const {
-    isOpen,
-    onTransitionEnd,
-    unstable__animationState: animated,
-  } = useMenuContext()
+  const { rootProps, ...rest } = props
+  const { isOpen, hasBeenOpened, onTransitionEnd } = useMenuContext()
 
   const listProps = useMenuList(rest, ref) as any
   const positionerProps = useMenuPositioner(rootProps)
 
   const styles = useMenuStyles()
 
+  const animation = useMemo(() => {
+    if (!hasBeenOpened.current) {
+      return undefined
+    }
+
+    return isOpen
+      ? `${enterAnim} 200ms cubic-bezier(0.4, 0, 0.2, 1.0)`
+      : `${exitAnim} 100ms ease-out`
+  }, [isOpen, hasBeenOpened])
+
   return (
     <chakra.div
       {...positionerProps}
-      __css={{ zIndex: props.zIndex ?? styles.list?.zIndex }}
+      __css={{
+        zIndex: props.zIndex ?? styles.list?.zIndex,
+      }}
     >
-      <MenuTransition
-        variants={motionVariants}
-        initial={false}
-        animate={isOpen ? "enter" : "exit"}
-        __css={{ outline: 0, ...styles.list }}
-        {...motionProps}
+      <chakra.div
+        __css={{
+          outline: 0,
+          visibility: isOpen ? "visible" : "hidden",
+          animation: animation,
+          ...styles.list,
+        }}
+        style={{
+          ...listProps.style,
+        }}
         className={cx("chakra-menu__menu-list", listProps.className)}
         {...listProps}
-        onUpdate={onTransitionEnd}
-        onAnimationComplete={callAll(
-          animated.onComplete,
-          listProps.onAnimationComplete,
-        )}
+        onAnimationEnd={callAll(onTransitionEnd, listProps.onAnimationComplete)}
```

Comment:
No need to call `animated.onComplete` anymore since CSS animation fires `animationend` event natively.